### DISCUSSION TITLE
Toonsquid current version

### DISCUSSION THREAD

**Discussion by Jebs88** [keiwando/toonsquid#634] [Q&A]
Can we still expect update on the current version, and can we still suggest to have some changes on it?

**Comment by Rohitgit2412** [comment]
Why do we need an update to Toonsquid version 1 when version 2 is already a major upgrade of it and will include all its features plus additional ones? I don't think we are getting any small update to the current version now. We will be getting a big upgrade directly. However you can still submit your suggestions and required features on the suggestion forum. Developer will surely look into it and try to bring it on future smaller updates of version 2.

**Reply by Jebs88** [reply]
I was curious since I didn't know if he'll be doing it like what Affinity did with their apps. But, thank you for the response.  Will be submitting suggestions too. Thanks.
